dockstore-webservice/src/main/java/io/openapi/api/impl/ToolsApiServiceImpl.java

```diff
@@ -567,6 +569,19 @@ private Response getFileByToolVersionID(String registryId, String versionIdParam
         return Response.status(status).build();
     }
 
+    public static List<Checksum> getFileDescriptorChecksumsForTrs(final SourceFile sourceFile) {
+        List<Checksum> trsChecksums = new ArrayList<>();
+        if (sourceFile.getChecksums() != null && !sourceFile.getChecksums().isEmpty()) {
+            sourceFile.getChecksums().stream().forEach(checksum -> {
+                Checksum trsChecksum = new Checksum();
+                trsChecksum.setType(DESCRIPTOR_FILE_SHA_TYPE_FOR_TRS);
```

Comment:
At this point it's not an issue, but what if we end up storing an MD5 checksum?

Reply:
yeah, I'd have to write a smarter converter. TRS wants the type to match a certain format, but when doing the checksum calculation in java the type doesn't match. Denis prefers we keep the java version as what we store in our db, and just do the conversion for what trs wants here.